Edit Module wording for Python Tab on Explicit Wait

JSheffield159 opened the pull request:
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.


<!--- Provide a general summary of your changes in the Title above -->

### Description
<!--- Describe your changes in detail -->
Hello this is my first PR in your project, but it is a simple text edit. Let me know if future PRs should be shorter or if this was a good size.

*Previously: 
from selenium.webdriver.support.ui import WebDriverWait
*Proposed:
from selenium.webdriver.support.wait import WebDriverWait

*Link to documentation for Python: 
https://www.selenium.dev/selenium/docs/api/py/webdriver_support/selenium.webdriver.support.wait.html#module-selenium.webdriver.support.wait
Python: selenium.webdriver.support.wait
*Link to documentation for Java: 
https://www.selenium.dev/selenium/docs/api/java/org/openqa/selenium/support/ui/WebDriverWait.html
Java: org.openqa.selenium.support.ui.WebDriverWait

### Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
Python utilizes a different import module then what was stated in Explicit Wait on two occasions. 
This is required to import the WebDriverWait class into Python.
### Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [x] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.
<!--- Provide a general summary of your changes in the Title above -->

JSheffield159 committed:
Edit Python Tabs for Explicit Wait

Lines 154 and 232
Python utilizes a different import module than what was stated in Explicit Wait twice.

*Previously: 
from selenium.webdriver.support.ui import WebDriverWait
*Proposed:
from selenium.webdriver.support.wait import WebDriverWait

*Link to documentation for Python: 
https://www.selenium.dev/selenium/docs/api/py/webdriver_support/selenium.webdriver.support.wait.html#module-selenium.webdriver.support.wait
Python: selenium.webdriver.support.wait
*Link to documentation for Java: 
https://www.selenium.dev/selenium/docs/api/java/org/openqa/selenium/support/ui/WebDriverWait.html
Java: org.openqa.selenium.support.ui.WebDriverWait

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/SeleniumHQ/seleniumhq.github.io?pullRequest=1135) <br/>All committers have signed the CLA.

diemol reviewed (changes requested):
Can you please make the change in the other waits.*.md files?

JSheffield159 committed:
Module wording waits.ja.md

Additional commit 1/3 for https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1135/commits/8528e9b3fb6ad70e6f5a5c46451baa13a3a1c691

JSheffield159 committed:
Module wording waits.pt-br.md

Additional commit 2/3 for https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1135/commits/8528e9b3fb6ad70e6f5a5c46451baa13a3a1c691

JSheffield159 committed:
Module wording waits.zh-cn.md

Additional commit 3/3 for https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1135/commits/8528e9b3fb6ad70e6f5a5c46451baa13a3a1c691

diemol committed:
Merge branch 'trunk' into patch-1

diemol reviewed (approved):
Thank you, @JSheffield159!
